selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/workspaces/CheckStoppingWsByTimeoutTest.java

```diff
@@ -0,0 +1,61 @@
+package org.eclipse.che.selenium.workspaces;
+
+import static org.testng.AssertJUnit.assertEquals;
+
+import com.google.inject.Inject;
+import java.util.concurrent.TimeUnit;
+import javax.inject.Named;
+import org.eclipse.che.api.core.model.workspace.Workspace;
+import org.eclipse.che.api.core.model.workspace.WorkspaceStatus;
+import org.eclipse.che.selenium.core.client.TestWorkspaceServiceClient;
+import org.eclipse.che.selenium.core.user.TestUser;
+import org.eclipse.che.selenium.core.utils.WaitUtils;
+import org.eclipse.che.selenium.core.workspace.TestWorkspace;
+import org.eclipse.che.selenium.pageobject.Ide;
+import org.eclipse.che.selenium.pageobject.ProjectExplorer;
+import org.eclipse.che.selenium.pageobject.ToastLoader;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class CheckStoppingWsByTimeoutTest {
+
+  private static int TOASTLOADER_WIDGET_LATECY_TIMEOUT_IN_MILLISEC = 20000;
+  @Inject private Ide ide;
+  @Inject private ProjectExplorer projectExplorer;
+  @Inject private ToastLoader toastLoader;
+  @Inject private TestWorkspaceServiceClient workspaceServiceClient;
+  @Inject private TestWorkspace testWorkspace;
+  @Inject private TestUser testUser;
+
+  @Inject
+  @Named("che.workspace_agent_dev_inactive_stop_timeout_ms")
+  private int stoppingTimeotInaciveWorkspace;
+
+  @Inject
+  @Named("active.state_activity_sheduler_period")
+  int shedulerRequstTimeout;
+
+  @BeforeClass
+  public void setUp() throws Exception {
+    int commonTimeout =
+        stoppingTimeotInaciveWorkspace
+            + shedulerRequstTimeout
+            + TOASTLOADER_WIDGET_LATECY_TIMEOUT_IN_MILLISEC;
```

Comment:
What about encapsulate 
```
int commonTimeout =
        stoppingTimeotInaciveWorkspace
             shedulerRequstTimeout
                 TOASTLOADER_WIDGET_LATECY_TIMEOUT_IN_MILLISEC;
```
to the private method, something like "getCommonTimeout"? This is not necesserely, but in my opinion it will be clearer:
```
public void setUp() throws Exception {
  ide.open(testWorkspace);
  projectExplorer.waitProjectExplorer();
  WaitUtils.sleepQuietly(getCommonTimeout(), TimeUnit.MILLISECONDS);
}
```

Reply:
And one more remark, typos "commonTimeout" -> "commonTimeout" ; "stoppingTimeotInaciveWorkspace" -> "stoppingTimeoutInactiveWorkspace"